css/css-ui/appearance-button-not-form-controls-001-ref.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Reference for CSS Basic User Interface Test: appearance: button on elements that are not form controls</title>
+<style>
+ #container { width: 500px; }
+</style>
+<div id="container">
+	<a>a</a>
```

Comment:
All elements in this container should be replaced with <button>, otherwise this test is a no-op.

Reply:
I intended for the test to use "mismatch". Using a button element will have different sizing so doesn't work.

Reply:
Ah, right, I see. But then the test is checking that at least one of these elements can be made to look like a button, not that all of them can. If you want to do this with a mismatch (which makes sense, given your comment above), you need to break this test case (and its ref) into a separate file for each element.